css/main.css

```diff
@@ -1,3 +1,83 @@
 /* Add your custom CSS here */
 
+@import url('https://fonts.googleapis.com/css?family=Sriracha&display=swap');
+
+body {
+	background-color: rgb(223, 223, 223);
+    font-family: 'Sriracha', cursive, sans-serif;
+    margin:0;
+}
+
+header {
+    width: 100vw;
+    margin-bottom: 0em;
+}
+header h1 {
+	text-align: center;
+}
+
+ul {
+    list-style: none;
+}
+
+.nav-bar a {
+	text-decoration: none;
+	color: rgb(54, 54, 54);
+}
+
+img {
+    width: 100vw;
+}
+
+
+.section-a h2, .section-b h2, .section-c h2 {
```

Comment:
You could create a new class called something like: `.centered-heading` and just apply that to the `h2` elements in the HTML.